src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildAWindowsDesktopProject.cs

```diff
@@ -341,6 +341,37 @@ public void SupportedOSPlatformVersionCanBeSetInDirectoryBuildTargets()
             GetPropertyValue(testAsset, "TargetPlatformMoniker").Should().Be("Windows,Version=10.0.19041.0");
         }
 
+        [WindowsOnlyTheory]
+        [InlineData("net5.0", true)]
+        [InlineData("net5.0-windows10.0.19041.0", true)]
+        [InlineData("netcoreapp3.1", false)]
```

Comment:
What is expected when netcoreapp3.1 is used along with UseWindowsForms?

Reply:
`SdkSupportedTargetPlatformIdentifier` will contain all the potential `TargetPlatformIdentifier`s that can be included in the TFM, so since netcore3.1 doesn't support any platforms in the TFM (`netcoreapp3.1-windows` is not valid) we expect this item group to be empty. 

For context, `SdkSupportedTargetPlatformIdentifier` will be used to populate the possible `TargetPlatformIdentifiers` in the project system's target framework drop downs